receiver/httpcheckreceiver/metadata.yaml

```
@@ -0,0 +1,34 @@
+name: httpcheckreceiver
+
+resource_attributes:
+
+attributes:
+  http.url:
+    description: Full HTTP request URL.
+    type: string
+  http.status_code:
+    description: HTTP response status code
+    type: int
+  http.method:
+    description: HTTP request method
+    type: string
+
+metrics:
+  httpcheck.success:
+    description: The response of the endpoint resulted in status 200.
+    enabled: true
+    sum:
+      value_type: int
+      aggregation: delta
+      monotonic: false
+    unit: 1
+    attributes: [http.url, http.status_code, http.method]
+  httpcheck.duration:
+    description: Measures the duration of HTTP check.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: false
+      value_type: int
+    unit: ms
+    attributes: [http.url]
```

```suggestion
  httpcheck.duration:
    description: Measures the duration of the HTTP check.
    enabled: true
    gauge:
      value_type: int
    unit: ms
    attributes: [http.url]
```

Suggestion is to make this a Gauge. Histogram feels like overkill. However, if we make this a Sum then the aggregation will not be meaningful.

There is an open issue discussing the (frequent) request for a synchronous Gauge instrument in the OpenTelemetry API. This kind of instrumentation is a perfect example of where this would be useful; when we get to that point, I expect that SDK views would be able to reconfigure a Gauge instrument as a Histogram instrument and vice versa -- as ways to increase or decrease the resolution of this metric. (Likely the Gauge histogram will fall out of of this; my goal is not to specify any of this until after OTel metrics reaches the 1.0 milestone.)